src/session/streams.js

```diff
@@ -1,5 +1,13 @@
 const StreamsSettings = require("../settings").of("streams")
 
+// TODO: Didn't seem to catch chat like this:
+// R >
+// [help]-GSIV:Illiphore: "being in an offensive stance help with hamstring success?"
```

Comment:
Do you have lnet set to go to familiar window?

Reply:
![lnet-thoughts](https://user-images.githubusercontent.com/1090434/86585027-c1a18400-bf53-11ea-91dc-42f7aa4edd7c.png)

Reply:
I do! My bad. Works perfectly. I'll remove the comment. We'll have to see how these two chat channels play out. They feel good to me combined for now but I'm curious if it will feel better to isolate them in the future.

Reply:
They both go into the same stream, the only real viable way to separate them will be hilites/regex because from the FE they are just `<pushStream id="thoughts">` components